pwndbg/wrappers/readelf.py

```diff
@@ -33,3 +34,49 @@ def _extract_jumps(line):
             return False
     except IndexError:
         return False
+
+@pwndbg.wrappers.OnlyWithCommand(cmd_name)
+def get_load_segment_info():
+    '''
+    Looks for LOAD sections by parsing the output of `readelf -l <binary>`
+    '''
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cmd = [get_jmpslots.cmd_path, "-l", local_path]
+    readelf_out = pwndbg.wrappers.call_cmd(cmd)
+
+    segments = []
+    load_found = False
+
+    # Output from readelf is 
+    # Type           Offset             VirtAddr           PhysAddr
+    #                FileSiz            MemSiz             Flags  Align
+    # LOAD           0x0000000000000000 0x0000000000000000 0x0000000000000000
+    #                0x0000000000000830 0x0000000000000830  R E    0x200000
+    # Account for this using two regular expressions
+    re_first = r"\s+LOAD\s+(0x[0-9A-Fa-f]+) (0x[0-9A-Fa-f]+) (0x[0-9A-Fa-f]+)"
+    re_secnd = r"\s+(0x[0-9A-Fa-f]+) (0x[0-9A-Fa-f]+)  (.)(.)(.)\s+(0x[0-9A-Fa-f]+)"
+    hex2int = lambda x: int(x, 16)
+
+    for l in readelf_out.splitlines():
+        if "LOAD" in l:
```

Comment:
1. LOAD might appear twice (see screenshot), is that okay?
https://i.gyazo.com/5deb4d216f1db4bd3c8e37fcc76cb71a.png

2. `LOAD` might appear in part of section name or interpreter path (yeah those edge cases...); maybe something like:
```
if line.lstrip().startswith('LOAD '):
``` 
(that's why parsing cli programs sucks on the long term, but it is okay for now; guess we don't want to reimlement readelf yet; and probably this has been done by [pwntools](https://pwntools.com) already, but we can't use it YET since they are not Python 3 compatible).

Reply:
`LOAD` appearing multiple times is okay and indended, yes. I like the `startswith` suggestion.